### ISSUE TITLE
https://cdn.ricin.im/linux/ is down

### ISSUE THREAD

**Issue by probonopd** [RicinApp/Ricin#165]
https://cdn.ricin.im/linux/ is down

**Comment by Monsterovich** [comment]
Download binaries.
https://www.mediafire.com/file/gkg9wck8sh7vhjx/Ricin-build.tar.gz

Also get tox libraries from here.
https://tox.chat/download.html#gnulinux

**Comment by probonopd** [comment]
It is still down.

**Comment by mkody** [comment]
The project is discontinued and I'm sure it will not come back.
Build it yourself following the doc if you want to use it anyway: https://github.com/RicinApp/Ricin#install

**EDIT: Actually, just look at the new website, the links are there: https://konv.im/downloads.html#linux**

**Comment by AyoungDukie** [comment]
Yes, I was going to point out @SkyzohKey has mentioned in other issues that the project has been migrated to Konv ([GitHub](https://github.com/SkyzohKey/Konv)) to develop the new version/successor to Ricin (I believe that he mentioned issues with code base quality, so he was essentially starting over).

Edit: just also realized that the full Ricin download history is on the new domain, so yeah, see @mkody 's answer for the solution.
